design-proposals/descheduler-support.md

```diff
@@ -0,0 +1,105 @@
+# Overview
+Deschedulers are Kubernetes components that act as load balancers for compute nodes.
+Their goal is to spread the workload in the cluster.
+
+The descheduler's way of moving Pods around is using the [eviction API](https://kubernetes.io/docs/concepts/scheduling-eviction/api-eviction/).
+Moving a regular pods is relatively simple, since they are supposed to be ephemeral.
+On the contrary, moving a `virt-launcher` pod that contains a running VM is more challenging, since VMs are stateful.
+
+Currently, there is an issue for deschedulers to work alongside KubeVirt because of the unique behavior of `virt-launcher` pods:
+
+From the descheduler's point of view, `virt-launcher` pods seem to resist eviction, but they actually migrate to another node.
+This causes the descheduler to make wrong decisions and take wrong actions that could destabilize the cluster.
```

Comment:
This needs more detailed explanation.  Specifically, the descheduler notes the failure to evict the virt-launcher pod and keeps trying to evict pods, typically resulting in it attempting to evict substantially all pods from the node.